utility/src/main/java/org/oppia/android/util/profile/CurrentUserProfileIdIntentDecorator.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.android.util.profile
+
+import android.content.Intent
+import org.oppia.android.app.model.ProfileId
+import org.oppia.android.util.extensions.getProtoExtra
+import org.oppia.android.util.extensions.putProtoExtra
+
+private const val PROFILE_ID_INTENT_DECORATOR = "PROFILE_ID_INTENT_DECORATOR"
```

Comment:
Suggest instead: "CurrentUserProfileIdIntentDecorator.profile_id_key". I think we're moving over toward this type of pattern for better uniqueness, and the screen name decorator isn't following it yet.

Reply:
Noted. Updated the same. This is just for the key itself right?